### ISSUE TITLE
Inheritance security rules violated

### ISSUE THREAD

**Issue by ClubWebware** [jamietre/CsQuery#150]
CsQuery sometimes gives the following error:

"Inheritance security rules violated while overriding member: 'CsQuery.Implementation.CombinedTextReader.CreateObjRef(System.Type)'. Security accessibility of the overriding method must match the security accessibility of the method being overriden."

Our web application needs to run in shared hosting environments, so our application must run under medium trust. Maybe that's part of the issue?

Any idea what might cause this error, and better yet: what to do about it?

Best regards, Marja
